src/apps/vpn/update/versionapi.cpp

```diff
@@ -29,7 +30,9 @@ VersionApi::~VersionApi() {
 }
 
 void VersionApi::start(Task* task) {
-  NetworkRequest* request = NetworkRequest::createForVersions(task);
+  NetworkRequest* request = NetworkRequest::create(task, 200);
+  request->get(
+      QString("%1/api/v1/vpn/versions").arg(AppConstants::apiBaseUrl()));
```

Comment:
I think it would be really helpful if all the Url Paths were in one place / file.

Ideally putting them all in one place:
* makes it easy to scan through the list
* means we're not repeating `.arg(AppConstants::apiBaseUrl())` everywhere.

Reply:
Good point!